src/Pages/Repositories/ContentListTable/components/SnapshotListModal/SnapshotListModal.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable quotes */
```

Comment:
i see a couple lint warnings, i think an `npm run format` would fix one and we might not need this line anymore?